rasterio/mask.py

```diff
@@ -110,7 +108,7 @@ def raster_geometry_mask(dataset, shapes, all_touched=False, invert=False,
 
 
 def mask(dataset, shapes, all_touched=False, invert=False, nodata=None,
-         filled=True, crop=False, pad=False, pad_width=0.5, indexes=None):
+         filled=True, crop=False, pad=False, pad_width=0.5, indexes=None, masked=True):
```

Comment:
@AntoineGuiot elsewhere in rasterio `masked=False` is the default, can you change this?

Reply:
And an addition to the docstring is needed, too.